tools/topology/topology2/production/tplg-targets-ace2.cmake

```diff
@@ -18,7 +18,7 @@ SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
 SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 
-"cavs-sdw\;sof-lnl-rt712-rt1320x1\;PLATFORM=lnl,SDW_DMIC=1,NUM_SDW_AMP_LINKS=2,\
+"cavs-sdw\;sof-lnl-rt712-l2-rt1320-l1\;PLATFORM=lnl,SDW_DMIC=1,NUM_SDW_AMP_LINKS=2,\
```

Comment:
I guess there's a matching kernel patch to updat ethe name and we haven't shipped this in sof-bin, so no problem to rename, right?

Reply:
Yes, the corresponding kernel PR is https://github.com/thesofproject/linux/pull/5223. And the tplg PR was just merged 3 days ago. https://github.com/thesofproject/sof/pull/9632

Reply:
@bardliao for CI, do you need this PR or kernel PR merged 1st ?

Reply:
We don't have any CI device with this configuration. So, it will not impact CI. However, I prefer merge kernel PR first to align the tplg name.

Reply:
We could merge this PR as https://github.com/thesofproject/linux/pull/5223 is merged.

Reply:
@kv2019i good for you ?

Reply:
Ack, was just waiting for the kernel part to be merged (which it is now).